packages/indexer-agent/src/db/migrations/12-add-scalar-tap-table.ts

```diff
@@ -0,0 +1,241 @@
+import { Logger } from '@graphprotocol/common-ts'
+import { QueryInterface, DataTypes } from 'sequelize'
+
+interface MigrationContext {
+  queryInterface: QueryInterface
+  logger: Logger
+}
+
+interface Context {
+  context: MigrationContext
+}
+
+export async function up({ context }: Context): Promise<void> {
```

Comment:
Following up on this comment, is this table definition in a model? The models should contain entire definition, so it's not split up between migration and model. Also the migration should really only be needed if the user already has data in the DB (not always the case).

Reply:
This isnt in the models so yes then this can be added in there too
To make it clear all tables in the migrations should be then also in the models right

Reply:
Migrations are just for changing data that's already in someone's DB, all table definitions should be in models.